src/libs/auth.js

```diff
@@ -259,25 +259,37 @@ authStore.subscribe(withErrorIgnoring(async (state, oldState) => {
 
 authStore.subscribe((state, oldState) => {
   if (state.nihStatus !== oldState.nihStatus) {
-    const notificationId = 'nih-link-warning'
     const now = Date.now()
     const expireTime = state.nihStatus && state.nihStatus.linkExpireTime * 1000
-    const expireStatus = Utils.cond(
-      [!expireTime, () => null],
-      [now >= expireTime, () => 'has expired'],
-      [now > expireTime - (1000 * 60 * 60 * 24), () => 'will expire soon']
-    )
-    if (expireStatus) {
+    const shouldNotify = expireTime && now > expireTime - (1000 * 60 * 60 * 24)
+    if (shouldNotify) {
+      const hasExpired = now >= expireTime
+      const notificationId = hasExpired ? 'nih-link-expired' : 'nih-link-expires-soon'
+      const expireStatus = hasExpired ? 'has expired' : 'will expire soon'
+
+      const muteNotificationPreferenceKey = `mute-nih-notification/${notificationId}`
+      const muteNotificationUntil = getLocalPref(muteNotificationPreferenceKey)
```

Comment:
I'm a bit confused about this variable. We don't start showing the notification after `muteNotificationUntil`. Isn't this the time when they pressed "Do not remind me again"?